src/Microsoft.AspNetCore.Sockets.Client/Connection.cs

```diff
@@ -194,17 +214,44 @@ public Task StartAsync(ITransportFactory transportFactory, HttpClient httpClient
                 var request = new HttpRequestMessage(HttpMethod.Options, url);
                 var response = await httpClient.SendAsync(request);
                 response.EnsureSuccessStatusCode();
-                var connectionId = await response.Content.ReadAsStringAsync();
-                logger.LogDebug("Connection Id: {0}", connectionId);
-                return connectionId;
+
+                var negotiationPayload = await response.Content.ReadAsStringAsync();
```

Comment:
Logging the response was the main concern and given the size of the response it's the simplest.